NangPaGo-api/NangPaGo-admin/src/main/java/com/mars/admin/domain/user/service/UserService.java

```diff
@@ -12,11 +16,16 @@
 @Service
 public class UserService {
 
-  private final UserRepository userRepository;
+    private final UserRepository userRepository;
+    private static int PAGESIZE = 10;
 
-  public UserResponseDto getCurrentUser(String email) {
-    return UserResponseDto.from(userRepository.findByEmail(email)
-        .orElseThrow(NPGExceptionType.NOT_FOUND_USER::of));
-  }
+    public UserResponseDto getCurrentUser(String email) {
+        return UserResponseDto.from(userRepository.findByEmail(email)
+            .orElseThrow(NPGExceptionType.NOT_FOUND_USER::of));
+    }
 
+    public Page<UserDto> getUserInfos(int page) {
```

Comment:
의견: `Infos` -> `List`

```suggestion
    public Page<UserDto> getUserList(int page) {
```